app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -2,15 +2,83 @@
   <div class="container">
     <div class="row">
       <h2>Sessions</h2>
-      <div> Coming soon - Sessions list!</div>
+    </div>
+    <template v-if="sessionsMetadata">
+      <template v-if="sessionsMetadata.length">
+        <div class="row fw-bold py-2">
+          <div class="col-3 session-col-header">Saved</div>
+          <div class="col-2 session-col-header">Label</div>
+          <div class="col-2 text-center session-col-header">Load</div>
+        </div>
+        <div class="row py-2" v-for="session in sessionsMetadata" :key="session.id">
+          <div class="col-3 session-col-value session-time">
+            {{formatDateTime(session.time)}}
+            <div v-if="isCurrentSession(session.id)" class="small text-muted">(current session)</div>
+          </div>
+          <div class="col-2 session-col-value session-label" :class="session.label ? '' : 'text-muted'">
+            {{session.label || "--no label--"}}
+          </div>
+          <div class="col-2 text-center session-col-value session-load">
+            <router-link v-if="isCurrentSession(session.id)" to="/">
+              <vue-feather class="inline-icon brand" type="home"></vue-feather>
+            </router-link>
+            <a v-else :href="sessionUrl(session.id)">
+              <vue-feather class="inline-icon brand" type="upload"></vue-feather>
+            </a>
+          </div>
+        </div>
+      </template>
+      <div id="empty-sessions" v-else>
+        No saved sessions yet.
+        <router-link class="wodin-link" to="/">Load the application</router-link> to create a new session.
```

Comment:
Some of these strings probably want pushing into userMessages? Or is this complicated within the template part of a component? This might make more sense as a maintenance ticket to do later (just thinking ahead to the inevitable internationalisation work)

Reply:
No, you're right, I just forgot about userMessages! Might as well get them in there.